samcli/commands/deploy/core/options.py

```diff
@@ -0,0 +1,87 @@
+"""
+Deploy Command Options related Datastructures for formatting.
+"""
+from typing import Dict, List
+
+from samcli.cli.row_modifiers import RowDefinition
+
+# The ordering of the option lists matter, they are the order in which options will be displayed.
+
+REQUIRED_OPTIONS: List[str] = ["stack_name", "capabilities", "resolve_s3"]
+
+# Can be used instead of the options in the first list
+SECONDARY_REQUIRED_OPTIONS: List[str] = ["guided"]
```

Comment:
Nice, I like it!

Reply:
I think I'm going to keep `Required Options` for consistency with other commands, and just change the `--guided` options to `Interactive Options`. Thoughts?

Reply:
Thats a good call out about consistency with the other help texts. I think this is a good move, but just a little concerned with the messaging about guided since if its provided, then the required options are no longer required. There is a description under the interaction options that mentions this, so this could be a moot point.